main.c

```diff
@@ -655,6 +656,9 @@ static int dev_added(struct tcmu_device *dev)
 	tcmu_dev_dbg(dev, "Got block_size %ld, size in bytes %lld\n",
 		     block_size, dev_size);
 
+	write_cache_enabled = tcmu_get_attribute(dev, "emulate_write_cache");
```

Comment:
write_cache is only an attribute in the kernel patch, so if a handler doesn't have implementation for the .reconfig, then it wont do anything. - None the less I will drop all the write_cache stuff for now and make it a separate patch.